src/utils/initUserConfig.ts

```diff
@@ -0,0 +1,39 @@
+import axios from 'axios';
+import { isBeta } from './common';
+import { initializeVisibilityFunctions } from './VisibilitySingleton';
+import createGetUserPermissions from '../auth/createGetUserPermissions';
+import { ChromeUser } from '@redhat-cloud-services/types';
+
+export type ChromeUserConfig = {
+  data: {
+    uiPreview: boolean;
+  };
+};
+
+export const initChromeUserConfig = async ({ getUser, token }: { getUser: () => Promise<ChromeUser>; token: string }) => {
+  const LOCAL_PREVIEW = localStorage.getItem('chrome:local-preview') === 'true';
+  let config: ChromeUserConfig;
+  if (!LOCAL_PREVIEW) {
```

Comment:
@Hyperkid123 we should probably also throw a `// FIXME` here once local preview is turned on permanently.